src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -3701,7 +3701,7 @@ compiler.warn.this.exposed.prematurely=\
 
 # 0: type
 compiler.err.generic.parameterization.with.value.type=\
-    Inferred type {0} involves generic parameterization by an inline type; This is allowed only with -XDallowGenericsOverValues
+    Inferred type {0} involves generic parameterization by an inline type
```

Comment:
Do you have examples when this can happen? Or is this a leftover (in which case I think Check.java could be cleaned up a bit)

Reply:
We had a discussion about possibly turning it into a throw. I think let us leave it in for now till the generics story is more evolved and we know where we are heading.

Reply:
No we don't know where this code could be exercised. All the inference scenarios are covered in the test case - which is why it had to be changed into positive test - So this could technically go away - only reason is to hold onto it to detect any surprises and to wait for generics story to crystalize.